packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -552,6 +569,8 @@ export async function toAbsoluteGlob(
   }
   root = globSafePath(root)
   const dir = importer ? globSafePath(dirname(importer)) : root
+  if (base && base.startsWith('./')) return pre + posix.join(dir, base, glob)
+  glob = base ? posix.join(base, glob) : glob
```

Comment:
Does this handle if `glob` is an alias? e.g. does it work with something like:

```js
import.meta.glob('@dir/*.js', { base: '/src/dir' })
```

Reply:
No, the base option is not necessary when setting alias, as the base path is set in vite.config.ts.

Reply:
I mean if you have that glob written with this file system:

```
src/
  dir/
    foo.js
    bar.js
package.json
```

And `@dir` maps to `src/dir`, shouldn't the glob be:

```js
Object.assign({
  "./foo.js": () => import("/src/dir/foo.js"),
  "./bar.js": () => import("/src/dir/bar.js"),
})
```

That's what I understand in the issue where the base path helps here. Or is this a different feature?

Reply:
So you can use the path specified in base as an alias. I was mistaken. 
I changed it to support aliases in [b07d814](https://github.com/vitejs/vite/pull/18510/commits/b07d814c850d11ee2eaa2839383e613f79b56040) because I think it is useful.

Reply:
I guess I was thinking of `base` differently as noted in https://github.com/vitejs/vite/pull/18510#discussion_r1827204126, hence it was confusing, but I'll take a look at the changes 👍